go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/cloudfoundry/socks5-proxy v0.0.0-20180530211953-3659db090cb2 // indirect
 	github.com/concourse/baggageclaim v1.6.6
 	github.com/concourse/dex v0.0.0-20200417202922-dcbe94f28c88
-	github.com/concourse/flag v1.0.0
+	github.com/concourse/flag v1.0.1-0.20200521134421-2be0b225f922
```

Comment:
I went ahead and made a new [release](https://github.com/concourse/flag/releases/tag/v1.1.0) of `concourse/flag`. You can now change this to reference `v1.1.0` directly without a commit SHA.